app/controllers/verify/finance_controller.rb

```diff
@@ -26,6 +27,20 @@ def create
 
     private
 
+    def submit_idv_form
+      result = idv_finance_form.submit(step_params)
+      analytics.track_event(Analytics::IDV_FINANCE_CONFIRMATION, result.to_h)
+
+      return if result.success?
+
+      render_failure
+      render_form
+    end
+
+    def form_valid_but_vendor_validation_failed?
```

Comment:
Why define this in the controllers instead of in the `IdvFailuerConcern` where it is invoked?